ruby/lib/ThreatExchange.rb

```diff
@@ -78,72 +53,62 @@ def threat_indicators(filter={})
       return data
     end
 
-    def indicator_pq(filter={})
-      begin
-        response = RestClient.get "#{@baseurl}/#{filter[:id]}/",
-        { params: { access_token: @access_token, fields: filter[:fields] } }
-        result = JSON.parse(response)
-        return result
-      rescue => e
-        puts e.inspect
-      end
+    def indicator_pq(params={})
+      get "#{@baseurl}/#{params[:id]}/", params
     end
 
-    def members()
-      begin
-        response = RestClient.get "#{@baseurl}/threat_exchange_members/",
-        { params: { access_token: @access_token } }
-        result = JSON.parse(response)
-        return result['data']
-      rescue => e
-        e.response
-      end
+    def members(params={})
+      get "#{@baseurl}/threat_exchange_members/", params
     end
 
-    def new_relation(data={})
-      data[:access_token] = @access_token
-      id = data.delete(:id)
-      begin
-        response = RestClient.post "#{@baseurl}/#{id}/related", data
-      rescue => e
-        e.inspect
-      end
+    def new_relation(params={})
+      id = params.delete(:id)
```

Comment:
Same as above, changing the passed in hash is bad form, especially for a library.